src/openrct2/audio/Audio.cpp

```diff
@@ -93,7 +94,18 @@ namespace OpenRCT2::Audio
                 }
             }
         }
-        LoadAudioObjects();
+        while (bool onPrepareDraw = true)
+        {
+            if (bool onDraw = false)
+            {
+                continue;
+            }
+            else
+            {
+                LoadAudioObjects();
+                break;
```

Comment:
So this will always happen

Reply:
yeah, I figured out the code did nothing 5 minutes ago